src/components/shared/Entity/EndpointConfigForm.tsx

```diff
@@ -56,6 +56,11 @@ function EndpointConfigForm({ endpointSchema }: Props) {
         output: categoryLikeSchema,
     });
 
+    const foo = (data: any) => {
+        console.log('>', data);
```

Comment:
Was this `console.log()` intentionally left behind?

Reply:
no